modules/swagger-codegen/src/main/resources/JavaSpringMVC/pom.mustache

```diff
@@ -121,10 +121,10 @@
     <swagger-core-version>1.5.8</swagger-core-version>
     <jetty-version>9.2.9.v20150224</jetty-version>
```

Comment:
I prepare a version of springboot so I do not use jetty and this was the version available in enterprise maven repository.